app/Http/Controllers/Api/MovieController.php

```diff
@@ -15,9 +15,17 @@ class MovieController extends Controller
      */
     public function index()
     {
-        return Movie::all();
+        if (request(['offset']) != null and request(['take']) != null ){
```

Comment:
ukoliko je offset koja stranica po redu onda stavi `page` u request-u umjesto `offset`, a ukoliko je `take`  koliko filmova po stranici stavi `perPage`

Reply:
Offset je ukupan broj filmova koje preskacem. Sa fronta se tako salje pa je lakse i citljivije. Sta mislis da ostavim ovako?

Reply:
ne bi trebao tako, treba da salje stranicu za koju treba da dobijes filmove i koliko filmova po stranici, takodje sobzirom da ces uvijek imati paginaciju, samo ce zavisiti da li ce to biti 100 filmova pa se moze desiti da bude samo jedna stranica, napravi default za to, znaci neka ti sa FE salje se default-no da ce biti recimo 50 filmova po stranici